src/modules/integrator-guide/pages/security-model/permissions-evaluation.adoc

```diff
@@ -0,0 +1,315 @@
+include::ROOT:partial$variables.adoc[]
+
+= Permission Evaluation
+
+The permission evaluation feature allows administrators to check which actions or operations can be performed by a user or a combination of roles.
+The evaluation entails comparing the different roles assigned to a user to determine the user's ability to perform the various operations within a resource.
+
+[NOTE]
+====
+xref:access-control/index.adoc[{PRODUCT_NAME} RBAC] uses the roles of the given user to determine if they have access to a given resource.
+====
+
+== How to evaluate applied permissions
+
+[NOTE]
+====
+Most of the {PRODUCT_NAME} resources allow you to define which xref:access-control/roles.adoc[roles] are allowed to access the resource.
+Locate the btn:[permissions] button in the user interface in order to manage the resource.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "security-model/access-control/general-permissions-button.png",
+    "alias": "general-permissions-button",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {
+    "x": 320,
+    "y": 0,
+    "h": 550,
+    "w": 1600
+  },
+  "focus": {
+    "x": 1466,
+    "y": 80,
+    "h": 31,
+    "w": 206
+  },
+  "annotations": [{
+    "kind": "box",
+    "x": 1536,
+    "y": 80,
+    "w": 134,
+    "h": 30
+  }]
+}
+----
+====
+
+1. Navigate to the resource of choice, and click the [btn:[permissions].
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "security-model/access-control/permission-button-page.png",
+    "alias": "new-user",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "kind": "box",
+    "padding": "xs",
+    "x": 1535,
+    "y": 75,
+    "w": 130,
+    "h": 42
+  }]
+}
+----
+
+2.Click the *[add +]* button.
+An evaluation window appears with two dropdown options, the *[select role(s)]* and *[search or select user]*.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "security-model/access-control/add-role.png",
+    "alias": "add-role",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "kind": "box",
+    "padding": "xs",
+    "x": 765,
+    "y": 150,
+    "w": 750,
+    "h": 80
+}]
+}
+----
+
+3.Click on the *[select roles]* or the *[select user]* dropdown and select a role or a user to evaluate.
```

Comment:
Use the UI macro; https://docs.antora.org/antora/latest/asciidoc/ui-macros/#button